com.unity.mobile.notifications/Runtime/Android/Plugins/Android/com/unity/androidnotifications/UnityNotificationRestartOnBootReceiver.java

```diff
@@ -0,0 +1,55 @@
+package com.unity.androidnotifications;
+
+// import com.unity3d.player;
+
+import android.app.PendingIntent;
+import android.content.BroadcastReceiver;
+import android.content.Context;
+import android.content.Intent;
+import android.support.annotation.Keep;
+import android.util.Log;
+
+import java.util.Calendar;
+import java.util.Date;
+import java.util.List;
+
+@Keep
+public class UnityNotificationRestartOnBootReceiver extends BroadcastReceiver {
+
+    @Override
+    public void onReceive(Context context, Intent received_intent) {
+        if (Intent.ACTION_BOOT_COMPLETED.equals(received_intent.getAction())) {
+
+            List<Intent> saved_notifications = UnityNotificationManager.LoadNotificationIntents(context);
+
+            for (Intent data_intent : saved_notifications) {
+
+                long fireTime = data_intent.getLongExtra("fireTime", 0L);
+                Date currentDate = Calendar.getInstance().getTime();
+                Date fireTimeDate=new Date(fireTime);
+
+                int id = data_intent.getIntExtra("id", -1);
+                boolean isRepeatable = data_intent.getLongExtra("repeatInterval", 0L) > 0;
+
+                if (fireTimeDate.after(currentDate) || isRepeatable) {
+
+                    // if (BuildConfig.DEBUG) {
```

Comment:
Not sure, but there is this com.unity3d.player.CPPConstants class, which has UNITY_DEVELOPMENT_PLAYER property, and luckily it's public, maybe you could use that. I imagine if we build to Android dev player, I would want these logs, while for Non Development, I don't

Reply:
Talked with Tomas offline about this.
1. com.unity3d.player.CPPConstants is private which can't be used here.
2. Once alternative is generating similar build config by our own in the post processor, which probably work for most of the cases. 
But this won't work for android studio, as these are two levels build config. 
3. The idea way is still making it work with BuildConfig.java, but I can't think of a way to do that without injecting the `import` code to the jave sources.

As a conclusion, we probably will ignore it for now.